Library/Formula/vim.rb

```diff
@@ -35,6 +35,15 @@ class Vim < Formula
   conflicts_with "ex-vi",
     :because => "vim and ex-vi both install bin/ex and bin/view"
 
+  # Default requirements mean installing the bottle here requires
+  # installing our Perl, Python & Ruby. To prevent that for everyone
+  # just pour the bottle when requirements already satisfied.
+  def pour_bottle?
+    (which("python") == HOMEBREW_PREFIX/"bin/python") &&
+      (which("ruby") == HOMEBREW_PREFIX/"bin/ruby") &&
+      Formula["perl"].installed?
+  end
```

Comment:
Hmm, are you sure this works how you expect (e.g. overriding the bottle `default_formula` behaviour)? If so I think it's probably an accident.